woocommerce/woocommerce-gateway-stripe
# Add Payment Methods and Settings Tabs to settings screen

# Changes proposed in this Pull Request:

Fixes #1707 

Description: Add two tabs to the new Stripe settings page and render a different component for each. Additionally, allow for a tab to be opened based on a URL query param.

# Testing instructions

1. Navigate to `http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe` and verify that you see two tabs.
2. Ensure that clicking on each tab shows that tabs content.
3. Add the `panel` query param with either `methods` or `settings` as values, like so:
`http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings`
4. Visit the URL from #3 and verify that the correct tab is automatically opened. Note: 'Payment Methods' is default tab.
5. Run `npm run test:js` to ensure tests pass.

-------------------
